packages/react/src/components/legacy/LegacyResponsiveTable/ResponsiveTable.stories.mdx

```diff
@@ -1,16 +1,16 @@
 import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
-import { ResponsiveTable } from './';
-import { TokensTable, Information } from '../../../../../docs-components';
-import { exampleRows } from './ExampleData';
-import classes from './Table.stories.module.css';
+import { LegacyResponsiveTable } from './';
+import { TokensTable, Information } from '../../../../../../docs-components';
+import { exampleRows } from '../../Table/ExampleData';
+import classes from '../../Table/Table.stories.module.css';
```

Comment:
Maybe just make a copy of these files as to not break the stories when we start refactoring `Table`. Since its mdx, the IDE won't pickup invalid paths until runtime.

Reply:
Will do 😊